w3lib/url.py

```diff
@@ -34,6 +34,8 @@ def _quote_byte(error):
 
 _safe_chars = RFC3986_RESERVED + RFC3986_UNRESERVED + EXTRA_SAFE_CHARS + b'%'
 
+remove_tab_newline = re.compile(r'[\t\n\r]')  # see https://url.spec.whatwg.org/#url-parsing
```

Comment:
I think it should be private. Also, this regexp can be used not only for removing tabs/newlines, but for e.g. fiinding them - what do you think about naming it e.g. _ascii_tab_newline_re, and adding a link to https://infra.spec.whatwg.org/#ascii-tab-or-newline?